src/middleware/commentAuthontication.js

```diff
@@ -0,0 +1,48 @@
+
+import  Jwt  from "jsonwebtoken";
+import Users from "../model/userModel";
+const commentsAuthorization = async (req, res, next) => {
+  let token;
+
+  try {
+    if (
+      req.headers.authorization &&
+      req.headers.authorization.startsWith("Bearer ")
+    ) {
+      token = req.headers.authorization.split(" ")[1];
+    }
+
+    if (!token) {
+      res.status(404).json({
+        status: "404",
+        message: "You Are Not Logged In Please login",
+      });
+    }
+
+    const decoded = await Jwt.verify(token, process.env.JWT_SECRET);
+    const logedUser = await Users.findById(decoded.id);
+
+    if (!logedUser) {
+      res.status(403).json({
+        status: "403",
+        message: "Token has Expired Please login Again",
+      });
+    }
+
+      req.Users = logedUser;
```

Comment:
You can name this to something like `req.loggedInUser = loggedInUser`

Reply:
Resolved naming from req.User = logedInUser to req.loggedInUser = loggedInUser